lib/PipelineC/PipelineC.cpp

```diff
@@ -254,8 +285,9 @@ _rp_manager_run_analysis(rp_manager *manager,
                          const char *step_name,
                          const char *analysis_name,
                          const rp_container_targets_map *target_map,
+                         const rp_string_map *options,
                          rp_invalidations *invalidations,
-                         const rp_string_map *options) {
+                         rp_error *error) {
```

Comment:
What was happening before this? Just returning a `nullptr`?

Reply:
Yes